app.json

```
@@ -44,6 +44,14 @@
 			"description": "Your SMTP Socket",
 			"value": "STARTTLS"
 		},
+		"MOBILESYNC_URL": {
+			"description": "MobileSync URL",
+			"value": "https://sync.example.com"
+		},
+		"MOBILESYNC_NAME": {
+			"description": "MobileSync Name",
+			"value": "https://sync.example.com"
```

Just to be consistent with the other default values provided
```suggestion
			"value": "sync.example.com"
```